src/gui/editors/PianoRoll.cpp

```diff
@@ -3456,10 +3610,30 @@ void PianoRoll::paintEvent(QPaintEvent * pe )
 		case ModeEditDetuning: cursor = s_toolOpen; break;
 	}
 	QPoint mousePosition = mapFromGlobal( QCursor::pos() );
-	if( cursor != NULL && mousePosition.y() > keyAreaTop() && mousePosition.x() > noteEditLeft())
+	if ( cursor != NULL && mousePosition.y() > keyAreaTop() && mousePosition.x() > noteEditLeft())
 	{
 		p.drawPixmap( mousePosition + QPoint( 8, 8 ), *cursor );
 	}
+	if (drawCustomLabels)
```

Comment:
Since this is the last drawn thing in `paintEvent`, this will draw over notes at the start of the pattern. Should it?